google-cloud-storage/test/google/cloud/storage/project_test.rb

```diff
@@ -235,12 +243,16 @@ def creds.is_a? target
 
   it "creates a bucket with logging bucket and prefix" do
     mock = Minitest::Mock.new
-    created_bucket = create_bucket_gapi bucket_name, logging: Google::Apis::StorageV1::Bucket::Logging.new(log_bucket: bucket_logging_bucket, log_object_prefix: bucket_logging_prefix)
+    created_bucket = create_bucket_gapi bucket_name,
```

Comment:
I don't see any changes in couple of these lines, it gets noisy while reviewing the actual change. Please revert the changes which do not contribute to this PR specifically.